index.html

```diff
@@ -10,7 +10,7 @@
  *
  * Indices
  *   - .toc for the Table of Contents (<ol class="toc">)
- *     -> <span class="secno"> for the section numbers
```

Comment:
did you mean this file to be included, doesn't have anything to do with docker?

Reply:
Yes it's important because dependency updates. In future changes I'll try to remove the generated files from mainstream or corner it under `docs/` or similar. For now it is how it is.